docs/source/getting_started/benchmarks/imagenette_benchmark.py

```diff
@@ -24,9 +24,11 @@
 | Moco          |        256 |    200 |              0.727 |   87.3 Min |      4.3 GByte |
 | NNCLR         |        256 |    200 |              0.726 |   86.8 Min |      4.2 GByte |
 | SimCLR        |        256 |    200 |              0.771 |   82.2 Min |      3.9 GByte |
+| SimMIM (**)   |        256 |    200 |              0.342 |   98.8 Min |     10.5 GByte |
```

Comment:
Why the ** instead of (ViT-B32)? We also use this format for the other models, for example DINO and MSN.

Reply:
(ViT-B32) is too long and destroys the format

Reply:
Please reformat the table in that case.

Reply:
Oh yes it makes sense